dist/Tie-File/lib/Tie/File.pm

```diff
@@ -862,14 +867,14 @@ sub _fill_offsets_to {
   return $self->{offsets}[$n] if $self->{eof};
 
   my $fh = $self->{fh};
-  local *OFF = $self->{offsets};
+  my $OFF = $self->{offsets};
   my $rec;
 
-  until ($#OFF >= $n) {
+  until (scalar @$OFF > $n) {
```

Comment:
It has been a while since I used $# I think the change from >= to > is appropriate based on the switch.

Reply:
Changed it to $#{$OFF}